banks/views.py

```diff
@@ -1,5 +1,33 @@
 from django.shortcuts import render
 from django.views.generic import ListView
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.views import LoginView as DjangoLoginView
+from .models import  Bank,Account
 
 # Create your views here.
 class BankListView(ListView):
+    model = Bank
+    template_name = 'bank_list.html'
+    context_object_name = 'banks'
+
+
+class AccountListView(LoginRequiredMixin,ListView):
+    model = Account
+    template_name = 'account_list.html'
+    context_object_name = 'accounts'
+
+    def get_queryset(self):
+        return Account.objects.filter(user=self.request.user)
+
+
+class LoginView(DjangoLoginView):
+    template_name = 'login.html'
+
+
+class UserAccountListView(LoginRequiredMixin, ListView):
```

Comment:
lets have option to 
1. update account (balance only)
2. delete account
3. create a new account (show all available branches)

Reply:
do this next PR